### PR TITLE
Cpf 3657/add external url prop to app config

### PR DESCRIPTION
**Pull request by yij793**
address:  https://nanopay.atlassian.net/browse/CPF-3657
note: add new property to appConfig in order to add external Url. Also change loginView backLink_ since it should not redirect to staging.ablii.com

### CODE REVIEW

File: src/foam/u2/view/LoginView.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ foam.CLASS({
       factory: function() {
         return !! this.model.backLink_ ?
           this.model.backLink_ :
-          !! this.appConfig.url ? this.appConfig.url : undefined;
+          !! this.appConfig.externalUrl ? this.appConfig.externalUrl : undefined;
```
Comment (jlhughes) [inline src/foam/u2/view/LoginView.js]: does this not work without the double negation !! ?  What bug are you protecting against with this?

Comment (yij793) [reply src/foam/u2/view/LoginView.js]: > does this not work without the double negation !! ? What bug are you protecting against with this?

I'm not sure. I did not write those code. I only changed url to externalUrl. I can rewrite the logic if you suggest.

Comment (kgrgreer) [reply src/foam/u2/view/LoginView.js]: You could also just do: this.appConfig.externalUrl || undefined;

Comment (jlhughes) [reply src/foam/u2/view/LoginView.js]: re: rewrite - yes please.  Update to Kevin's recommendation.  Thanks.

Comment (jlhughes) [inline src/foam/u2/view/LoginView.js]:
```suggestion
          this.appConfig.externalUrl : undefined;
```

File: src/foam/u2/view/LoginView.js [hunk 2]
```diff
@@ -191,9 +191,7 @@ foam.CLASS({
       class: 'String',
       name: 'backLink_',
       factory: function() {
-        return !! this.model.backLink_ ?
-          this.model.backLink_ :
-          !! this.appConfig.url ? this.appConfig.url : undefined;
+        return this.model.backLink_ ? this.model.backLink_ : this.appConfig.externalUrl || undefined;
```
Comment (kgrgreer) [inline src/foam/u2/view/LoginView.js]: Could be simplified as just:
this.model.backLink_ || this.appConfig.externalUrl || undefined;

**Pull request merged**